File: ICMS/app/src/main/java/com/example/ahmedpc/icms/MainLauncherActivity.java
package com.example.ahmedpc.icms;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;
import android.widget.TextView;

public class MainLauncherActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_launcher);
       ImageView imageView = findViewById(R.id.icms_iv_logo);
        Animation animation = AnimationUtils.loadAnimation(getApplicationContext(),R.anim.fadelogo);
        imageView.startAnimation(animation);
        logolauncher logolauncher = new logolauncher();
        logolauncher.start();


    }

    private class logolauncher extends Thread {

        public void run() {
            try {
                sleep(3000);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
            Intent intent = new Intent(MainLauncherActivity.this, CatureImageActivity.class);
            startActivity(intent);
            MainLauncherActivity.this.finish();
        }
    }
}

File: Foodna/app/src/main/java/com/example/ahmedpc/foodna/MainLauncherActivity.kt
package com.example.ahmedpc.foodna

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainLauncherActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_launcher)

        val btn_signin = findViewById<Button>(R.id.foodna_btn_signin)
        val btn_signup = findViewById<Button>(R.id.foodna_btn_signup)
        val btn_skip = findViewById<Button>(R.id.foodna_btn_skip)

        btn_signin.setOnClickListener {
            val Intent = Intent(this@MainLauncherActivity, SignInActivity::class.java)
            startActivity(Intent)
        }


        btn_signup.setOnClickListener {
            val Intent = Intent(this@MainLauncherActivity, SignUpActivity::class.java)
            startActivity(Intent)
        }


        btn_skip.setOnClickListener {
            val Intent = Intent(this@MainLauncherActivity, LocationActivity::class.java)
            startActivity(Intent)
        }


    }
}
